Apps/W1/EDocument/app/src/Document/EDocuments.Page.al

```diff
@@ -55,6 +56,28 @@ page 6122 "E-Documents"
                 }
             }
         }
+        area(factboxes)
+        {
+#if not CLEAN25
+            part("Attached Documents"; "Document Attachment Factbox")
```

Comment:
Why are we introducing an obsolete part?

Since this hasn't existed, no one can depend on the old attachment. We can just introduce the new part from the get-go?

Reply:
This is defined in the task to add them both. I'm not sure why it was decided like this, but the plan is to clarify this part during demo.